templates/profile/profileInformation.html.twig

```diff
@@ -0,0 +1,18 @@
+<div class="row justify-content-center mb-3">
+    {% if user.image is null %}
```

Comment:
Depending on `user.image`, define some `imageUrl` var and put it in your `img` tag. That will make your `if-else` smaller and easier to maintain as you won't be need to change both tags.

Reply:
Resolve